test_nvf_direct_multiple_fd.py

```diff
@@ -0,0 +1,114 @@
+# torchrun --local-ranks-filter=0 --nnodes 1 --nproc-per-node 2 test_nvf_direct_multiple_fd.py
+
+# TODO: Disabled for AssertionError: Cannot import nvfuser_direct if nvfuser module is already imported.
```

Comment:
Will this AssertionError stay or is this only short-term assertion for testing? IIRC, for now, plan was to only use `nvfuser_direct` for multi-device path.